docker/install/ubuntu_install_papi.sh

```diff
@@ -23,11 +23,15 @@ set -o pipefail
 apt-get update --fix-missing
 
 # deps
-apt-get install -y linux-tools-common linux-tools-generic
+apt-get install -y linux-tools-common linux-tools-generic kmod
 
 cd /
 git clone https://bitbucket.org/icl/papi.git
-cd papi/src
+# Pulling the latest version of this has broken the images before. Checkout the commit below for now.
+cd papi
+git checkout e3c263b6fb578695085d0ab093e4fd1ee5aeac23
```

Comment:
did we want to pull a specific version rather than a sha1 that looks random to a new user? it would simplify install instructions.

can you also update the docs in docs/install/from_source.rst to include this version constraint?

Reply:
Papi is not mentioned in the `from_source` file. I added the comment [here](https://github.com/apache/tvm/blob/ffb5fa47edd5fec443bfafcdd886f1f2a925810d/docs/how_to/profile/papi.rst) instead.